LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultClassCDevicesMessageSender.cs

```diff
@@ -51,7 +51,7 @@ public async Task<bool> SendAsync(IReceivedLoRaCloudToDeviceMessage cloudToDevic
                     return false;
                 }
 
-                if (!RegionFactory.TryGetRegion(loRaDevice.LoRaRegion, out var region))
+                if (!RegionManager.TranslateToRegion(loRaDevice.LoRaRegion, out var region))
```

Comment:
If it is a bool TranslateToRegion(param1, out param2) it should be `TryTranslateToRegion` imo. It can fail (Region.NotSet is used for instance)